app/models/user.js

```diff
@@ -0,0 +1,10 @@
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define('users', {
+    ID: { type: DataTypes.INTEGER, unique: true, primaryKey: true, allowNull: false },
+    name: { type: DataTypes.STRING, allowNull: false },
+    lastName: { type: DataTypes.STRING, allowNull: false },
+    gmail: { type: DataTypes.STRING, unique: true, allowNull: false },
```

Comment:
Why gmail? It seems to me that email is more generic.

Reply:
Sorry, I don't happened to my